github.go

```diff
@@ -54,37 +56,57 @@ func runGitHubHandler(c *gin.Context) {
 
 	importer.Token, err = importer.getToken(i)
 	if err != nil {
-		c.Error(err)
-		c.JSON(http.StatusBadRequest, fmt.Sprintf("Unable to get token. %+v", err))
+		ginutil.WriteProblemError(c, err, problem.Response{
+			Type: "prob/provider/github/getting-token-error",
+			Title: "Error getting token.",
+			Status: http.StatusBadRequest,
+			Detail: "Unable to get token.",
```

Comment:
Suggest adding some details here about what token, such as

```suggestion
			Detail: fmt.Sprintf("Unable to get token by ID %d or create token new token.", i.TokenID),
```

Reply:
Same comment for the other problems. The problem detail should include lots of THIS-PROBLEM-INSTANCE-related data, to make it easier to understand what's going on.

Like, the problem detail should be able to explain the error itself, as the error field is not actually part of the IETF RFC-7807 spec.

So almost include as much info as possible, as long as you're not including any confidential or sensitive data (such as the token value)

Reply:
Understood.

I will try to include more info for the errors tomorrow. I need to go through the code a bit more to understand the specifics.

Reply:
I wrote a lot over at https://github.com/iver-wharf/wharf-provider-azuredevops/pull/14

Instead of repeating myself, try to use that one as a template to how to solve stuff for this one